scripts/commands/helm.sh

```diff
@@ -32,7 +32,7 @@ _trap_hook() {
         _idx=1
         while test "$_idx" -lt "$_max"; do
             f="$(awk "BEGIN{FS=\"\x00\"}{print \$$_idx}" "${decrypted_file_list}")"
-            _idx="$(expr "$_idx" + 1)"
+            _idx=$((_idx+1))
```

Comment:
yeah, see line 241

Reply:
either way, some tests still failing ... shame ...
Nice CI you got here.

Reply:
obviously I could probably pass the tests with ...

```
        if xargs --help >/dev/null 2>&1; then
            if [ "${QUIET}" = "false" ]; then
                echo >&2
                # shellcheck disable=SC2016
                xargs -0 -n1 sh -c 'rm "$1" && printf "[helm-secrets] Removed: %s\n" "$1"' sh >&2 <"${decrypted_file_list}"
            else
                xargs -0 rm >&2 <"${decrypted_file_list}"
            fi
      else
[awk-variant]
      fi
```

but that wouldn't be fair ... or should implement specific unit tests ...

IDK.

In my case, I'll probably stick with my initial fix, and replace the printf "%s\0" by a regular echo.
But now I get why upstream should stick with that trick ... Nicely done.